README.md

```diff
@@ -80,41 +80,15 @@
 
 **[]** is a...
 
-## 🛠 Built With <a name="built-with"></a>
-
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>tech 1</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">HTML</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>tech 2</summary>
-  <ul>
-    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>tech 3</summary>
-  <ul>
-    <li><a href="https://www.javascripttutorial.net/">JavaScript</a></li>
-  </ul>
-</details>
 
 
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
+- **[used modules]**
+- **[arrow functions]**
 - **[key_feature_3]**
 
```

Comment:
- [ ] OPTIONAL
Generally, the subheading text Author1 should be replaced with your full name in your ReadMe.